<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCrop</class>
 <widget class="QMainWindow" name="QCrop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCrop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/qcrop/images/window.png</normaloff>:/qcrop/images/window.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Image</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="actionStartConversion"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="actionStartConversion"/>
   <addaction name="separator"/>
   <addaction name="actionE_xit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPhotoView" name="images"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionE_xit">
   <property name="icon">
    <iconset resource="qcrop.qrc">
     <normaloff>:/qcrop/images/exit.png</normaloff>:/qcrop/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionStartConversion">
   <property name="icon">
    <iconset resource="qcrop.qrc">
     <normaloff>:/qcrop/images/convert.png</normaloff>:/qcrop/images/convert.png</iconset>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="qcrop.qrc">
     <normaloff>:/qcrop/images/filenew.png</normaloff>:/qcrop/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="qcrop.qrc">
     <normaloff>:/qcrop/images/fileopen.png</normaloff>:/qcrop/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPhotoView</class>
   <extends>QTreeView</extends>
   <header>qphotoview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qcrop.qrc"/>
 </resources>
 <connections/>
</ui>
